services/generic/color_control.md

```diff
@@ -1,45 +1,104 @@
-### Color control service
+# Color Control Service
 
-The service has to be used to control color components of a lightning device.
+The service can be used to control color components of a lighting device, such as LED strips or bulbs.
 
-#### Service names
+## Service name
 
 `color_ctrl`
 
-#### Interfaces
+## Interfaces
 
-Type | Interface                  | Value type |  Description
------|----------------------------|------------|-------------------
-in   | cmd.color.get_report       | null       | The command is a request for a map of color component values
-in   | cmd.color.set              | int_map    | value is a map of color components. val= {"red":200, "green":100, "blue":45}
-out  | evt.color.report           | int_map    | Map of color components, where value is component intensity.
--|||
-in   | cmd.color.start_transition | object     | Example value: val= {"component": "red", "transition": "up", "duration": 127}
-in   | cmd.color.stop_transition  | str        | Stop fading/enhancing single color component. val = "red"
+| Type | Interface                  | Value type | Description                                                                                                                                                      |
+|------|----------------------------|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| in   | cmd.color.get_report       | null       | Requests a map of color component values.                                                                                                                        |
+| in   | cmd.color.set              | int_map    | Sets values for all color components using a map in which key is a [`color_component`](#definitions) listed in [`sup_components`](#service-properties) property. |
+| out  | evt.color.report           | int_map    | Returns map of color components, where value is the component intensity.                                                                                         |
```

Comment:
Actually when tested on Fibaro RGBW2 Controller, this will send only one color per every frame received by RGBW2 Controller.
To clearify, this RGBW2 Controller sends 3 frames in a row (one by one).
Zwave-ad sends then 3 consecutive evt.color.report's - each of them informing about intensity of different color.

Maybe it does not need to be included in this documentation, as it permits that kind of situations.
Especcialy if storage-sub_value is used along with it.

Reply:
This is a big problem.

Reply:
Discussed, ZW needs to cache color components and send report with all.